cmd/debug.go

```diff
@@ -109,8 +103,17 @@ func debugInfoCmd(ctx *cli.Context) error {
 	if !loggedIn {
 		fmt.Fprintf(w, "%s\t%v\n", "Logged In", loggedIn)
 	} else {
-		fmt.Fprintf(w, "%s\t%s <%s>\n", "Identity", self.Body.Name, self.Body.Email)
-		fmt.Fprintf(w, "%s\t%s\n", "Username", self.Body.Username)
+		fmt.Fprintf(w, "%s\t%s\n", "Session Type", session.Type())
+
+		if session.Type() == apitypes.MachineSession {
+			fmt.Fprintf(w, "%s\t%s\n", "Machine ID", session.ID())
+			fmt.Fprintf(w, "%s\t%s\n", "Machine Token ID", session.AuthID())
+		} else {
+			fmt.Fprintf(w, "%s\t%s\n", "User ID", session.ID())
+			fmt.Fprintf(w, "%s\t%s <%s>\n", "Identity", session.Name(), session.Email())
+		}
+
+		fmt.Fprintf(w, "%s\t%s\n", "Username", session.Username())
```

Comment:
We need a common "concept" because this is the identity slug, so we could just say `Identity`.